waveorder/stokes.py

```diff
@@ -0,0 +1,344 @@
+"""
+Overview
+--------
+
+This module collects Stokes- and Mueller-related calculations. 
+
+The functions are organized into three groups:
+
+1) A forward function group: 
+
+A = A_matrix(swing, scheme="5-State")
```

Comment:
We should use a more descriptive name here: `A_matrix` should be called `Instrument_matrix`, `Sys_matrix`, `I2Stokes_matrix`.

Reply:
I just renamed `A_matrix` to `I2S_matrix` and added a new `S2I_matrix` function. 

I'm open to more iteration on these names.